packages/dashboard/src/components/util/dateTimeUtil.tsx

```diff
@@ -0,0 +1,7 @@
+// Format Date object to get date and time to display MM/DD/YY HH:MM:SS format
+export const getFormattedDateTime = (rawDate: Date) => {
```

Comment:
Mind checking if we use a date formatter already ?

Reply:
yes, using in react-components but not with same format. here added this to get in MM/DD/YY HH:MM:SS